### ISSUE TITLE
Black screen sometimes

### ISSUE THREAD

**Issue by zhangpeng23796** [openstf/stf#915]
When I use stf , always it works very vell, but it may be black screen sometime.
The console output as follows:
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Requesting frame producer to stop
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Disconnecting from minicap service
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Stopping minicap service
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Sending SIGTERM to minicap
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (jni/minicap/minicap.cpp:197) Received SIGTERM, stopping"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (jni/minicap/minicap.cpp:516) Closing client connection"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:291) Destroying virtual display"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Keeping 627x867 as current frame producer projection
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Requesting frame producer to start
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Launching screen service
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "PID: 12690"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: Using projection 1080x1920@488x867/0"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:241) Creating SurfaceComposerClient"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:244) Performing SurfaceComposerClient init check"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:255) Creating virtual display"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:261) Creating buffer queue"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:264) Setting buffer options"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:268) Creating CPU consumer"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:272) Creating frame waiter"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (external/MY_minicap/src/minicap_26.cpp:276) Publishing virtual display"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Connecting to minicap service
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (jni/minicap/JpgEncoder.cpp:64) Allocating 6268932 bytes for JPG encoder"
INF/device:plugins:screen:stream 19027 [GWY0217117012398] Reading minicap banner
INF/device:plugins:screen:stream 19027 [GWY0217117012398] minicap says: "INFO: (jni/minicap/minicap.cpp:448) New client connection"
IMP/device:plugins:connect 19027 [GWY0217117012398] Remote Connect Started for device "GWY0217117012398" at "10.167.3.124:7401"

![image](https://user-images.githubusercontent.com/26616673/42795934-871623e4-89b9-11e8-84d4-b6b1155a22a9.png)

**Comment by sorccu** [comment]
Please don’t remove the issue template, we need the details it asks you to provide.

Which device is this?

**Comment by zhangpeng23796** [comment]
The device is HUA WEI Mate9，the model is MHA-AL00.

**Comment by HeartIsBeat** [comment]
When I wanted to enter password in HUA WEI P20， it show black screen on stf.

**Comment by Zoe1808** [comment]
Try to turn off the Security Input Settings in HUAWEI. See [here](https://github.com/openstf/stf/issues/876#issuecomment-421909403)

**Comment by HeartIsBeat** [comment]
It's this? But it still show black screen.
![image](https://user-images.githubusercontent.com/20472976/45684611-71d4ab80-bb79-11e8-972c-5993b8ba6c7a.png)

**Comment by ngbrito** [comment]
Hello @HeartIsBeat ,

Did you fix this issue? I'm facing the same issue.

Tx

**Comment by HeartIsBeat** [comment]
@ngbrito No.But I can still enter password when screen is black. After press 'enter', you can see screen shows and apk tips "username or password is wrong~"
However Huawei 10 can't do this.

**Comment by XiaLeon** [comment]
@HeartIsBeat Hi, I met the same issue on Huawei device. I don't thing the password page is a problem because I think this is about security and user privacy. Turning into black screen is not a problem for this.
But what I met is that the other pages except password page are black screen. 
Did you have the same issue? what did you mean that "Huawei 10 can't do this"? which device is yours? what model is it ? can you please let me know ? 
Thank you so much!

**Comment by XiaLeon** [comment]
@zhangpeng23796 Hi, Can you please let me know what tool you are using? Is there a link or instruction guide? Thanks

**Comment by HeartIsBeat** [comment]
@XiaLeon Not the same issue.
"Huawei 10 can't do this"  means  I can't enter password when screen is black on Huawei 10 maybe. I don't have Huawei 10 (COL-AL10)  now.
